src/modules/multi-tenancy/tenant.service.ts

```diff
@@ -3,20 +3,12 @@ import { createConnection, EntityManager, getConnectionManager } from 'typeorm';
 import { ShortenedNamingStrategy } from '@medusajs/medusa/dist/utils/naming-strategy';
 import { ConfigModule } from './types';
 import { MedusaRequest } from '../../core';
-import TenantRepository from './tenant.repository';
-
-type ConstructorParams = {
-	tenantRepository: typeof TenantRepository;
-};
 
 @Service()
 export class TenantService {
 	static readonly resolutionKey = 'tenantService';
 
-	readonly #tenantRepository: typeof TenantRepository;
-
-	constructor(private readonly container: ConstructorParams, private readonly config: ConfigModule) {
-		this.#tenantRepository = container.tenantRepository;
+	constructor(private readonly config: ConfigModule) {
```

Comment:
The configModule is the second argument of the construtcor. The first one is the cradle, this is how medusa works under the hood